sys-boot/grub/grub-9999.ebuild

```diff
@@ -119,6 +122,19 @@ QA_PRESTRIPPED="
 "
 
 pkg_pretend() {
+	if [[ ${GRUB_PLATFORMS} =~ "arm64" ]] ; then
+		ebegin "Checking for aarch64-cros-linux-gnu-gcc"
+		if type -p aarch64-cros-linux-gnu-gcc > /dev/null ; then
```

Comment:
Ok, this way requires the overlay is already set, but at the end of the day the effect is the same.
Fix commited.